src/o365/spfx/commands/project/project-externalize/model/ExternalizeEntry.ts

```diff
@@ -0,0 +1,6 @@
+export interface ExternalizeEntry {
```

Comment:
Since this interface shares all properties with the `External` interface, would it make sense to have it inherit from `External` instead of duplicating the code?

Reply:
good idea, fixed, thanks